packages/edit-site/src/components/header/navigation-toggle/index.js

```diff
@@ -35,7 +35,7 @@ function NavigationToggle( { icon, isOpen, onClick } ) {
 		return null;
 	}
 
-	let buttonIcon = <Icon size="32px" icon={ wordpress } />;
+	let buttonIcon = <Icon size="36px" icon={ wordpress } />;
```

Comment:
Why do we use `36px` here? We have `32px` in the design. Or is that obsolete?

Reply:
The Post Editor uses a 36px icon, I'd rather keep them consistent. 🙂